public/main.js

```diff
@@ -1,20 +1,37 @@
 $(document).on('ready', function(){
+    var name = '';
 	submitAnswer();
     handleMessage();
     displayLeaders();
+    onlineUsers();
```

Comment:
inconsistent functions' names. In one case you call `displayLeaders` in another `onlineUsers`. First function name starts with verb, second with noun. In both cases they do very similar thing.